include/object_fields.h

```diff
@@ -33,6 +33,7 @@
 #define OBJECT_FIELD_CHAIN_SEGMENT(index) rawData.asChainSegment[index]
 #define OBJECT_FIELD_OBJ(index)           rawData.asObject[index]
 #define OBJECT_FIELD_SURFACE(index)       rawData.asSurface[index]
+#define OBJECT_FIELD_PAINTING(index)      rawData.asPainting[index]
```

Comment:
i think this macro is a waste of time, void pointer should be perfectly fine for this purpose. doesnt need a whole new object field type

Reply:
do this